web/annotation_editor_layer_builder.css

```diff
@@ -965,7 +997,144 @@
     }
 
     .editToolbar {
+      --editor-toolbar-colorpicker-arrow-image: url(images/toolbarButton-menuArrow.svg);
```

Comment:
Please add this file to the following list:
https://github.com/mozilla/pdf.js/blob/319e85449ea6b49bb89a5542def5f88894c812ee/gulpfile.mjs#L1077-L1082